server/routerlicious/packages/services-utils/src/morganLoggerMiddleware.ts

```diff
@@ -45,10 +45,10 @@ export function jsonMorganLoggerMiddleware(
                 additionalProperties = computeAdditionalProperties(tokens, req, res);
             }
             const properties = {
-                [HttpProperties.method]: tokens.method(req, res),
+                [HttpProperties.method]: tokens.method(req, res) ?? "METHOD_UNAVAILABLE",
                 [HttpProperties.pathCategory]: `${req.baseUrl}${req.route?.path ?? "PATH_UNAVAILABLE"}`,
                 [HttpProperties.url]: tokens.url(req, res),
-                [HttpProperties.status]: tokens.status(req, res),
+                [HttpProperties.status]: tokens.status(req, res) ?? "STATUS_UNAVAILABLE",
```

Comment:
I think it is safer to use `||` here. `??` works only when the preceding variable is `undefined`, if variable is empty string, it will not use the chained value. Empty string will be `falsy` so `||` would work safer here. 

Check the example of `const c` below:
![image](https://user-images.githubusercontent.com/3719246/183908455-21328cd8-cc1c-4f8e-bd0e-a12dc24a8000.png)

Reply:
I was thinking that there is no situation where the values here are empty string, but I agree, it is probably better to do it the way you suggested. Let me send a new iteration. I will leave `pathCategory` using `??` though, because maybe there could be a `req.route?.path` that is empty?

Reply:
yes, I think path make sense, we could have dumb path route that doing get "/api/" lol